### ISSUE TITLE
SOC death.

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#27703]
(WEB REPORT BY: westbend REMOTE: 206.221.180.138:7777)
# Revision
3a8ea2ca751944bfd4184889193d249cbe1612e0

# Description
SOCed myself in the mouth as wizard for a martian meme. Instead of changing (or dying for that matter) it changed me to a catbeast and instantly pushed me out of my body, leaving the body catatonic.
# Steps to Reproduce
SOC in mouth of Synergy Podbay with Martian Setting.
# What you Expected
Becoming a martian.
# What Actually Happened
Got pushed out of my new body and ghosted permanently.

**Comment by Dacendeth** [comment]
being made a catbeast instead of a martain can be it's missfire chance it has, I wonder if the force ghost is related to that.

**Comment by boy2mantwicethefam** [comment]
Right, so I dug up a little and did some experimentation and here's what I came up with
- Staff of changing yourself works fine
- Staff of changing yourself while honorbound as a no-guns wizard works fine
- Staff of changing yourself into a martian works fine
- Staff of changing yourself enough times for the 10% chance to misfire to kick in and end up with a random other option works fine

I dug up your particular case and Postmorph is the only culprit since it handles mind transferrence, and in the case you are somehow registered without a mind it's supposed to assign your ckey to the mob. Whatever bug affected you is probably very rare or very situational since I can't really see what broke it for you.
